src/dealDashboard/dealMenubar/dealMenubar.scss

```diff
@@ -1,36 +1,48 @@
 @import "src/resources/elements/primeDesignSystem/variables";
 
 deal-menubar {
-  display: grid;
-  grid-template-columns: auto auto auto;
-  align-items: center;
+  display: flex;
   justify-content: end;
-  grid-column-gap: 10px;
-  grid-row-gap: 20px;
 
-  i.fas {
-    font-size: 20px;
+  .dealActionButtons {
+    display: grid;
```

Comment:
grid for 
[Edit Cancel]

Note, after this fix, might make sense to not move grid here in the first place, but I didn't want to do tooo many changes